src/swoole/Swoole/Coroutine/System.php

```diff
@@ -32,7 +34,7 @@ public static function dnsLookup(string $domain, float $timeout = 5)
      * @param string $command [required]
      * @param bool $get_error_stream [optional] = false
```

Comment:
might be helpful to add a `@return array` here

Reply:
Will do

Reply:
The method returns string|false